src/containers/BooksForm.js

```diff
@@ -0,0 +1,64 @@
+import React, {Component} from 'react';
+import {connect}          from 'react-redux';
+import { createBook } from '../actions/index';
+
+export const CATEGORIES = ["Action", "Biography", "History", "Horror", "Kids", "Learning", "Sci-Fi"];
+
+const mapDispatchToProps = dispatch => {
+  return {
+    createBook: (book) => {
+      dispatch(createBook(book))
+    }
+  };
+};
+
+
+class BooksForm extends Component {
+
+  state = {title: '', category: ''};
```

Comment:
What is it used for?

Reply:
It just a state that picks up the form inputs, and sends those inputs to the redux state via mapTo....

Reply:
Yeah but I couldn't find where you guys used it. Can you point me to the part of codebase?

Reply:
[Line 21](https://github.com/inhaq/StoreCMS/blob/458246ba6fa93167f7db8ed4410ddd5aee441344/src/containers/BooksForm.js#L21)
[Line36](https://github.com/inhaq/StoreCMS/blob/458246ba6fa93167f7db8ed4410ddd5aee441344/src/containers/BooksForm.js#L36)
😄

Reply:
[Line 21](https://github.com/inhaq/StoreCMS/blob/458246ba6fa93167f7db8ed4410ddd5aee441344/src/containers/BooksForm.js#L21)
[Line 36](https://github.com/inhaq/StoreCMS/blob/458246ba6fa93167f7db8ed4410ddd5aee441344/src/containers/BooksForm.js#L36)
😄